17.md

```diff
@@ -0,0 +1,130 @@
+
+```js
+{
+  "id": "<usual hash>",
+  "pubkey": "<sender-pubkey>",
+  "created_at": now(),
+  "kind": 14,
+  "tags": [ 
+    ["p", "<receiver-1-pubkey>", "<relay-url>"],
+    ["p", "<receiver-2-pubkey>", "<relay-url>"],
+    ["e", "<kind-14-id>", "<relay-url>", "reply"] // if this is a reply
+    ["subject", "<conversation-title>"],
+    ...
+  ],
+  "content": "<message-in-plain-text>",
+}
+```
+
+The content MUST follow the same plaintext rules of `kind:1`. Tags that mention, quote and assemble threading structures MUST follow [NIP-10](10.md). Fields `id` and `created_at` are required. 
+
+The set of `pubkey` + `p` tags defines a group chat. Clients SHOULD render messages of the same group in a continuous thread. If a new `p` tag is added or the current one is removed, a new group is created. 
+
+An optional `subject` tag defines the current name/topic of the conversation. Any member can change the topic by simply submitting a new `subject` to an existing `pubkey` + `p`-tags group. There is no need to send `subject` in every message. The newest `subject` in the thread is the subject of the conversation. 
+
+The use of other tags is highly encouraged (e.g. `content-warning` ([NIP-36](36.md)) if the content is NSFW). 
+
+## Encrypting and Broadcasting
+
+Following [NIP-59](59.md), the unsigned `kind:14` chat message must be sealed (`kind:13`) and then gift wrapped (`kind:1059`) to each receiver and the sender individually:
+
+```js
+{
+  "id": "<usual hash>",
+  "pubkey": randomPublicKey,
+  "created_at": randomTimeUpTo1WeekInThePast(), 
+  "kind": 1059, // gift wrap
+  "tags": [
+    ["p", receiverPublicKey, "<relay-url>"] // receiver
+  ],
+  "content": nip44Encrypt(
+    {
+      "id": "<usual hash>",
+      "pubkey": senderPublicKey,
+      "created_at": randomTimeUpTo1WeekInThePast(), 
+      "kind": 13, // seal
+      "tags": [], // no tags
+      "content": nip44Encrypt(unsignedKind14, senderPrivateKey, receiverPublicKey),
+      "sig": "<signed by senderPrivateKey>"
+    },
+    randomPrivateKey, receiverPublicKey
+  ),
+  "sig": "<signed by randomPrivateKey>"
+}
+```
+
+The inner `kind:14` event MUST never be signed. The encryption algorithm MUST use the latest version of [NIP-44](44.md). 
+
+Clients MUST verify if pubkey of the `kind:13` is the same pubkey on the `kind:14`, otherwise any sender can impersonate others by simply changing the pubkey on `kind:14`.
+
+Clients SHOULD randomize `created_at` in up to a week in the past in both the seal and the wrap to make sure grouping by `created_at` doesn't reveal the group's membership information. 
+
+The gift wrap's `p`-tag can be the receiver's main pubkey or an alias key created to receive DMs without exposing the receiver.
+
+Clients SHOULD only broadcast giftwrap events to the `read` relays of each recipient's [NIP-65](65.md).
```

Comment:
This is a fatal flaw and should be changed.

I've talked about this many times with @vitorpamplona and @staab and others, and everybody always agrees we should use some other relay list, but this is still in the NIP and you're still using it? What is going on?

Reply:
Hum.. we never talked about nip 65 being a flaw at all. So, I am not sure what you mean. Regardless, this NIP is working well with the read relays of nip 65.

Reply:
No, we talked about allowing users to specify a restricted set of (possibly just one even) relays that are used only for receiving DMs and not anything else, and DM clients being able to fetch a users's entire DM set from that single relay which would be very private and efficient.

Reply:
Sure! I just added the language to follow NIP-65 directions, whatever they are.